01-JS-DOM-APIs/Exercise6/js/main.js

```diff
@@ -0,0 +1,38 @@
+window.onload = function(){
+    document.getElementById('btn').addEventListener("click", tableGenerator);
+
+    function tableGenerator(){
+    
+    let clean = document.getElementById('table');
```

Comment:
You're getting the same element twice. DOM operations are expensive so it's better to perform as less as possible.